src/map/skill.hpp

```diff
@@ -442,6 +442,13 @@ enum e_skill_blown	{
 	BLOWN_TARGET_BASILICA		= 0x20, // If target is in Basilica area
 };
 
+// Enum for skill_dance_overlap*
```

Comment:
```suggestion
// Enum for skill_dance_overlap flag parameter
```

Reply:
We were using the * syntax to clarify it's used for all functions that start with that string. It's also used in other places like that. But I'll just push your suggestion because I have no opinion on it either way.